### ISSUE TITLE
InputfieldImage should always be open on "image upload" modal

### ISSUE THREAD

**Issue by BernhardBaumrock** [ryancramerdesign/ProcessWire#1467]
if you have a ckeditor field and click on the image button you can directly upload images to an InputfieldImage field. but if that field is set to CLOSED (ajax loading in my case) this field will also be closed in the modal after clicking "upload image" in the modal.

i think it would be more user friendly to have this field always open in the modal

**Comment by cb2004** [comment]
Yeah great idea, and I posted in the forums about this a couple of months ago. That way you can hide the image field from the page but it is visible when using it in the RTE.

**Comment by ryancramerdesign** [comment]
Thanks @BernhardBaumrock I have added this enhancement.

@cb2004 I did have to omit the "hidden" (and "locked" fields) fields because I don't think we can safely make the assumption that a field is hidden for this purpose. Personally I use hidden fields for values I want to only be API accessible, and I suspect others do the same. Also, if we allowed uploads to a hidden field, there would be no way from the UI to delete them. :)

**Comment by cb2004** [comment]
@ryancramerdesign that's a very good point right there :)

**Comment by isellsoap** [comment]
@BernhardBaumrock Could you please close this issue? Thank you!
